forge.config.ts

```diff
@@ -25,7 +27,75 @@ function getPlatformSpecificResources() {
   return [path.join('./resources/', getPlatform(), getArch())]
 }
 
+function getPostMakeHook() {
+  // we use the hook function only on windows to sign the binary so in
+  // all other cases we just return an empty object
+  if (getPlatform() !== 'win') {
+    return (forgeConfig: ForgeConfig, options: ForgeMakeResult[]) =>
+      Promise.resolve(options)
+  }
+
+  return async (forgeConfig: ForgeConfig, options: ForgeMakeResult[]) => {
```

Comment:
I realize that this is a pretty nitpicky comment, but it should probably be `makeResults` and not `options` 🤔